bochs/configure.ac

```diff
@@ -1252,6 +1252,29 @@ if test "$support_cet" -gt 0 -a "$use_x86_64" = 0; then
   AC_MSG_ERROR([Control Flow Enforcement Technology feature emulation require x86-64 enabled])
 fi
 
+support_uintr=0
+AC_MSG_CHECKING(for User Level Interrupt support)
+AC_ARG_ENABLE(cet,
+  AS_HELP_STRING([--enable-uintr], [for User Level Interrupt support (no)]),
+  [if test "$enableval" = yes; then
+    AC_MSG_RESULT(yes)
+    AC_DEFINE(BX_SUPPORT_UINTR, 1)
+    support_cet=1
+   elif test "$enableval" = no; then
+    AC_MSG_RESULT(no)
+    AC_DEFINE(BX_SUPPORT_UINTR, 0)
+   fi
+   ],
+  [
+    AC_MSG_RESULT(no)
+    AC_DEFINE(BX_SUPPORT_UINTR, 0)
+  ]
+  )
+
+if test "$support_uintr" -gt 0 -a "$use_x86_64" = 0; then
+  AC_MSG_ERROR([User Level Interrupt feature emulation require x86-64 enabled])
```

Comment:
require**s**?
I guess it was copied from another description...

Reply:
Really required. The UINTR support ONLY on processor with X86_64 and ONLY in long64 mode.
There are two ways to enforce that.
I started from first one by adding everywhere 
`#if BX_SUPPORT_UINTR && BX_SUPPORT_X86_64`
but turned so ugly so I going away from that and prefer to enforce it in configure script + compile time assert in config.h

Reply:
I meant a bit different matter: the phrase is about a single feature/single technology emulation, so the **-s** ending seems to be required for verbs in this context. I.e. "CET feature emulation require**s** x86-64 enabled", "UINTR feature emulation require**s** x86-64 enabled" and so on. Am I wrong here?

Sorry for grammar comments instead of technical ones.